src/main/java/org/example/application/DeleteCustomerService.java

```diff
@@ -1,15 +1,18 @@
 package org.example.application;
 
+import lombok.AllArgsConstructor;
 import org.example.domain.customer.CustomerId;
 import org.example.port.in.customer.DeleteCustomerUseCase;
 import org.example.port.out.CustomerRepository;
 
+@AllArgsConstructor
+
 public class DeleteCustomerService implements DeleteCustomerUseCase {
     private CustomerRepository customerRepository;
     private DeleteInvoiceService deleteInvoiceService;
```

Comment:
i znow tu powinno byc tylko "gadanie" z adapterami zew

Reply:
??? - I thought that if we delete the customer from our accounting office - all of the invoices that belong to this particular user should be deleted as well- not to keep the memory in our program.

Reply:
Okay resolved.